themes/classic/templates/catalog/product.tpl

```diff
@@ -57,6 +57,7 @@
         {block name='page_content_container'}
           <section class="page-content" id="content">
             {block name='page_content'}
+              <!-- @todo: use include file='catalog/_partials/product-flags.tpl'} -->
```

Comment:
In order to reduce the number of changes so close to RC, I prefer to perform the refactoring in next version